src/Podimo.ConstEmbed/ConstEmbedGenerator.cs

```diff
@@ -23,7 +23,7 @@ public void Initialize(IncrementalGeneratorInitializationContext context)
                     var @class = pair.Right.GetAdditionalFileMetadata(pair.Left, "ConstEmbed");
                     return (Class: @class, File: pair.Left);
                 })
-                .Where(static pair => pair.Class is not null);
+                .Where(static pair => pair.Class is not null and not "");
```

Comment:
This is just a nitpick, but I prefer the built in `string` methods.

```suggestion
                .Where(static pair => !string.IsNullOrWhiteSpace(pair.Class));
```

Reply:
I wasn't sure, but amended to use `string.IsNullOrEmpty()` instead.

There is also potentially another fix that may be nicer than this check, I didn't do it because an empty string doesn't make much sense to begin with for a class name, but it might provide better diagnostics in the case something else has gone wrong:

```cs
    public static string? GetAdditionalFileMetadata(
        this AnalyzerConfigOptionsProvider provider,
        AdditionalText file,
        string name)
    {
        bool succeed = provider.GetOptions(file)
            .TryGetValue(
                $"build_metadata.AdditionalFiles.{name}",
                out var value
            );
        if (succeed)
            return value;
        else
            return true;
    }
```

Reply:
I think the current fix is fine, although it would be nice to have some more diagnostics. In any case, I'll resolve and merge these changes. Thanks for the help! 😄